src/pages/ModuleEditor/ModuleEditor.tsx

```diff
@@ -70,7 +70,12 @@ export default function ModuleEditor({match: {params: {moduleId}}}: Props) {
   }
 
   async function onTagInput(input: string) {
-    setTagSuggestions(await getTags(input));
+    try {
+      setTagSuggestions(await getTags(input));
+    }
+    catch (e) {
+      setTagSuggestions(tagSuggestions);
+    }
```

Comment:
@jasekiw do you have a better suggestion on how to handle if the tag suggestion search returns a 400?

Reply:
I would suggest you don't do the search if there is no input to search off of. 
```
  setTagSuggestions(input.trim() === '' ? [] : await getTags(input));
```